spec/formatting.md

```diff
@@ -85,10 +85,16 @@ At a minimum, it includes:
 
 Implementations MAY include additional fields in their _formatting context_.
 
-## Expression and Markup Resolution
+## Resolved Values
 
-_Expressions_ are used in _declarations_, _selectors_, and _patterns_.
-_Markup_ is only used in _patterns_.
+This specification allows for the same value to be used for:
+- formatting in a _placeholder_,
+- selection in a _selector_ _expression_,
+- as the _operand_ of another _expression_, or
```

Comment:
I think we want to mention declarations explicitly. I think "another" can just be "an" (another presumes there is already an expression...)

```suggestion
- as the _operand_ of an _expression_ (including of a _declaration_), or
```

Reply:
Why call out declarations as anything special here? Exactly the same sort of processing is done for example in both of the following equivalent messages:
```
.local $foo = {42}
.local $bar = {$foo :number}
{{The answer is {$bar}}}
```

```
.local $foo = {42}
{{The answer is {$foo :number}}}
```

Reply:
The reason is so that declarations are mentioned explicitly. When I made the mental checklist of "places expressions can appear", declarations were missing. This also serves as a reminder of how an expression gets to be the value of an option or operand (as we don't permit nesting).

Reply:
+1

Reply:
```suggestion
- as the _operand_ of another _expression_ (including _local declarations_), or
```